packages/app/src/Application.js

```diff
@@ -52,6 +52,8 @@ export default class Application
      *  for devices with dual graphics card **webgl only**
      * @param {boolean} [options.sharedTicker=false] - `true` to use PIXI.Ticker.shared, `false` to create new ticker.
      * @param {boolean} [options.sharedLoader=false] - `true` to use PIXI.Loaders.shared, `false` to create new Loader.
+     * @param {PIXI.Container} {options.stage} - Pass existing stage or container
+     * @param {number} {options.animationDeltaMax} - How many frames will be processed by animation if user switches the tab
```

Comment:
55-56 missing dots :)

Reply:
ok, I think this can be addressed as separate PR, we can add this to CI or somewhere